npm/webpack-dev-server/test/makeWebpackConfig.spec.ts

```diff
@@ -24,12 +23,18 @@ describe('makeWebpackConfig', () => {
           progress: true,
           overlay: true, // This will be overridden by makeWebpackConfig.ts
         },
+        optimization: {
+          noEmitOnErrors: true, // This will be overridden by makeWebpackConfig.ts
+        },
       },
       devServerEvents: new EventEmitter(),
     }
     const actual = await makeWebpackConfig({
       devServerConfig,
-      sourceWebpackModulesResult: sourceDefaultWebpackDependencies(devServerConfig),
+      sourceWebpackModulesResult: createModuleMatrixResult({
```

Comment:
Updated this to use `createModuleMatrixResult`, because we claim to be testing `webpack-dev-server` v3 in this test, but the version of Webpack was getting resolved to v5, when it should be v4.